<ui version="4.0" >
 <class>ContaminantDialog</class>
 <widget class="QDialog" name="ContaminantDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Contaminants</string>
  </property>
  <widget class="QPushButton" name="removeButton" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>500</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>500</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="_contaminantTable" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>40</y>
     <width>741</width>
     <height>231</height>
    </rect>
   </property>
   <property name="rowCount" >
    <number>0</number>
   </property>
   <property name="columnCount" >
    <number>6</number>
   </property>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLineEdit" name="_name" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>280</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>280</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Compund Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>10</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Contaminants</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_hlaw" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>310</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>310</y>
     <width>257</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Henrys law constant [cm3-water/cm3-vapor]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>340</y>
     <width>79</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Koc [g-i/g-oc]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_koc" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>340</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_diffAir" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>370</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>370</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Diffusivity in air [cm2/s]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>400</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Diffusivity in water [cm2/s]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="_diffWater" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>400</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_atmoConc" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>430</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>430</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Atmospheric concentration [g/cm3]</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="okCancelButton" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>500</y>
     <width>156</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelButton</sender>
   <signal>accepted()</signal>
   <receiver>ContaminantDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>357</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelButton</sender>
   <signal>rejected()</signal>
   <receiver>ContaminantDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>357</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
